common/lib/xmodule/xmodule/modulestore/mongo/base.py

```diff
@@ -1958,16 +1958,24 @@ def ensure_indexes(self):
                 ('_id.name', pymongo.ASCENDING),
                 ('_id.revision', pymongo.ASCENDING),
             ],
-            background=True)
+            # While this index would more safely be "background=True", it's not currently created that way.
```

Comment:
@macdiesel I was hesitant to catch *all* exceptions (at least all the general `OperationFailure` exceptions) and it wasn't simple to detect only an exception caused by the index already existing. But what you suggest would work - I'll dig into the MongoDB source to see if I can find an exact error code.